### DISCUSSION TITLE
Maximum distance Yolo V8 medium size can detect objects

### DISCUSSION THREAD

**Discussion by Habibi6010** [ultralytics/ultralytics#3235] [General]
Hello every one

I want to know how can i calculate the maximum distance from a camera Yolo v8 medium size can detect objects. I know it depends on the camera feature size of the object and Yolo anchor and so on, but I am looking to find a formula for this one.

For example, if I know the size of the phone in the real world is 10 cm , how far I can detect it with a webcam?

**Comment by glenn-jocher** [comment]
@Habibi6010 hello!

The maximum detection distance for an object using YOLOv8 (or any object detection model) isn't determined by a straightforward formula, as it depends on various factors including the resolution of your camera, the object's size, the model's architecture, and the quality of the training data.

For a given real-world object size, such as a phone that's 10 cm, the detection distance will depend on how many pixels that object occupies in the image. If the phone becomes too small in the image (i.e., it occupies very few pixels), it may fall below the model's detection threshold.

To estimate the maximum distance, you'd need to consider:
- The camera's field of view (FOV).
- The camera's resolution.
- The minimum number of pixels required for YOLOv8 to reliably detect the object.

As a starting point, you can experiment by placing the object at various distances from the camera and noting at what distance the model stops detecting the object reliably. This can give you a practical sense of your system's limitations.

For more detailed guidance on model performance and capabilities, you can refer to our documentation at https://docs.ultralytics.com. 😊
